UmatiServerLib/BindVariable.hpp

```diff
@@ -114,8 +116,8 @@ class BindVariable {
     if constexpr (std::is_enum<T>::value) {
       return getToVariantFuncForEnumArray(value);
     } else if constexpr (
-      std::is_class<T>::value &&
```

Comment:
@ccvca : Can you look at this change?

Reply:
This should not be necessary, as LocalizedText is handled as a structured data type.